image.go

```diff
@@ -12,33 +13,49 @@ import (
 	"os"
 
 	"github.com/disintegration/gift"
+	"golang.org/x/image/bmp"
 )
 
-func resizeAndRotate(img image.Image, width, height int) image.Image {
-	g := gift.New(
-		gift.Resize(width, height, gift.LanczosResampling),
-		//gift.UnsharpMask(1, 1, 0),
-		gift.Rotate180(),
-	)
+func resizeAndRotate(img image.Image, width, height int, devname string) image.Image {
+	g, _ := deviceSpecifics(devname, width, height)
```

Comment:
Set `g` to the return value of the new function created below. The returned error is unused right now, but I should fix that.